src/pages/SignUp/SignUp.css

```diff
@@ -0,0 +1,77 @@
+.signup-wrapper {
+    margin-top: 7.4rem;
+    margin-left: 1rem;
```

Comment:
margin-left has been added twice here

Reply:
The values used for margin/padding and z-index differ from project to project so that's why I am not using it in the component library.
And I wanted to make the videos line in the center.
And done removed extra margin-left.